arches/app/media/js/viewmodels/map.js

```diff
@@ -479,6 +479,48 @@ define([
             });
         };
 
+        this.beforeMove = function(e) {
+            e.cancelDrop = (e.sourceParent!==e.targetParent);
+        };
+
+        this.reorderOverlays = function(e) {
+            const mapOrder = ko.observableArray(e.sourceParent());
+            const newOrder = self.createNewOverlayOrder(mapOrder);
+            self.sendNewOverlayOrder(newOrder)
+        };
+
+        this.keyDownHandler = function (context, e) {
+            // reorder list in the front-end by only using keyboard inputs
+            var li = $(this);
```

Comment:
No need for jQuery here as far as I can tell. You could just use `this` and then change `li[0]` to `li` later in `moveOverlays`